src/r5py/r5/transport_network.py

```diff
@@ -5,7 +5,6 @@
 
 
 import json
-import os.path
 import pathlib
```

Comment:
Import as `from pathlib import Path` ?

Reply:
I know many people like to do these abbreviated imports, and also numpy as np and pandas as pd, but I always feel uneasy about those things. Code should be as explicit as possible, so I don’t have to go to the top of the file to figure out where something came from. My personal convention there is to only do absolute imports, except for resources from within the same package.

That said, of course you can outvote me on that, and I’ll accept it ;)

Reply:
Well I guess this a bit of a question of taste / what one is used to do, and there are two different schools to this as you mention. Personally, I prefer having more clean / shorter code which to me is more readable, and that can be achieved with abbreviated imports. However, I'm okay going either way, as long as the style is systematic. So you can decide :)

Reply:
Yes, with the java paths it definitely makes sense to be as explicit as possible. So let's aim to follow the same style throughout the project.